<launch>
  <!--node name="serial_node1" pkg="rosserial_python" type="serial_node.py" output="screen">
    <param name="port" value="/dev/ttyACM0" />
    <param name="baud" value="115200" />
  </node-->
  <node pkg="imu_filter_madgwick" type="imu_filter_node" name="imu_filter_node"/>
  <node pkg="marin_controller" type="controller_node" name="controller_node"/>
  <node pkg="marin_controller" type="robot_state_node" name="robot_state_node"/>
  <node pkg="marin_controller" type="odometry_publisher_node" name="odometry_publisher_node"/>
</launch>
